cpu.go

```diff
@@ -254,14 +258,25 @@ func (ca *Cagent) CPUWatcher() *CPUWatcher {
 
 	stat.UtilAvg.SetDurationsMinutes(durations...)
 	stat.UtilAvg.mu.Unlock()
+	ca.cpuWatcher = &stat
+
+	if len(ca.CPUUtilTypes) > 0 && len(ca.CPUUtilDataGather) > 0 || len(ca.CPULoadDataGather) > 0 {
```

Comment:
Please comment what the if condition is.
If the explanation is the comment on line 264, please move it before the `if`

Reply:
moved the comment